assets/babel.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    // When no targets are specified: Babel will assume you are targeting the oldest browsers possible.
```

Comment:
This is invalid JSON, though I guess Babel allows it? "Oldest browsers possible" def isn't accurate with how this is set up, so I think we should delete.